test/scenarios/attestation/input/test-scenarios/testDataExport.yaml

```diff
@@ -7,9 +7,8 @@ testScenarios:
   - description: FakedTestScenario
     steps:
       - exampleFile: ../examples/Create_AttestationProvider.json
+        step: create attestation provider
       - exampleFile: ../examples/Update_AttestationProvider.json
-        replace:
-          - pathInBody: tags.Property1
-            to: $(storageAccountId)
```

Comment:
Don't quite get idea of test scenario definition. but why do we remove the replace part? Also, should we define a different step name for different step?

Reply:
this is used originally to test the resource replacement logic here: https://github.com/Azure/autorest.az/pull/838/files#diff-5af11d6df026b8335c645b37062bdbdd1f6691a0b92dac7557e871405a699d05L1459
In latest oav, it will replace example parameters internally. So we delete this logic and data in codegen.